Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -830,24 +817,25 @@ class DropDownMenu
     return item;
   }
 
-  private static OnItemClick (this: HTMLAnchorElement, ev: MouseEvent): boolean
+  private static OnItemClick (anchor: HTMLAnchorElement, ev: MouseEvent, clickHandler: Nullable<Function>): boolean
   {
-    ev.preventDefault();
-
-    if (this.href == null || this.href === '')
+    if (anchor.href == null || anchor.href === '')
       return false;
 
     DropDownMenu._itemClicked = true;
     DropDownMenu.ClosePopUp(true);
+
+    var result = true;
     try
     {
-      eval(this.getAttribute('javascript')!);
+      if (clickHandler != null)
```

Comment:
```suggestion
      if (clickHandler)
```

Reply:
@a-ctor I'll go with the `!==` check for consistency with existing code.